assets/constants.ts

```diff
@@ -5891,5 +5891,8 @@ export default {
   ],
 
   "DEV_MODE": true,
-  "EMULATOR_MODE": true
+  "EMULATOR_MODE": true,
+  "ASSET_TYPE_AUDIO": "audio",
```

Comment:
Per Sean's suggestion, we can move these constants to be moved in a separate PR. 
I have created an [issue](https://github.com/oppia/oppia/issues/13299) for the same.
Can you handle these changes in a PR for that issue ?
(That way both of us will be able to use these constants)
@anandwana001 WDYT ?

Reply:
Let me send a separate PR now, and we will merge and use it in our individual PRs.

Reply:
but I think it will take more time as a different reviewer has to review and then again we have to wait, so for now we can go with this and will resolve the merge conflicts at the end while merging, this might be more time saving.

Reply:
Sounds good to me.

Reply:
Can we have the change moved to the separate PR ? 
Because, it is best if we get these changes (related to constants) merged as soon as possible -- as it will be required for subtopics and story node migrations as well. WDYT ?